tests/phpunit/tests/rest-api/rest-global-styles-controller.php

```diff
@@ -455,4 +460,41 @@ public function test_get_item_schema() {
 		$this->assertArrayHasKey( 'settings', $properties, 'Schema properties array does not have "settings" key' );
 		$this->assertArrayHasKey( 'title', $properties, 'Schema properties array does not have "title" key' );
 	}
+
+	
+	public function test_get_theme_items() {
+		wp_set_current_user( self::$admin_id );
+		switch_theme( 'theme1' );
```

Comment:
I'm going to need some help here :). I noticed that there's a "theme1" theme in php unit test fixtures, I tried using it here but failed (see unit test failure). I'm not really sure how we do switch to that particular theme in a php unit test in core.

Reply:
It should work, see:

https://github.com/WordPress/wordpress-develop/blob/26c6b33b870a7630202b317a515d9d6e9183c258/tests/phpunit/tests/theme/wpThemeJsonResolver.php#L59-L63

Unless some set up work is missing like there:

https://github.com/WordPress/wordpress-develop/blob/26c6b33b870a7630202b317a515d9d6e9183c258/tests/phpunit/tests/theme/wpThemeJsonResolver.php#L15-L22

Reply:
I see that there are two themes living at `tests/phpunit/data/themedir1`: `block-theme` and `block-theme-child`. To use any of those we need to register the `themedir1` as a theme root (so tests look fore themes in that directory as well). That's what the the resolver tests do (the link Greg shared).